### ISSUE TITLE
[OJ] OJ benchmark not refreshed

### ISSUE THREAD

**Issue by dequeueing** [hezean/sustc#102]
### Related module

Benchmark

### What happened

Hi. I modified my code and in local benchmark, it seemed that I had passed all the points in deleteVideo(). No debugs or wrong answers displayed in the log.

![image](https://github.com/hezean/sustc/assets/127177415/85ab2704-144d-40f4-9539-75e7a845e07a)

![image](https://github.com/hezean/sustc/assets/127177415/05ffbcf1-0355-4545-862c-3566cf326097)
![image](https://github.com/hezean/sustc/assets/127177415/88df76c3-08e7-43c8-bb0a-4f280219857f)

In short, I get correct result in local testbench, but not in OJ. I ensure that the uploaded jar is the correct version.

Thanks.

### What you expected to happen

OJ grade get updated

**Comment by hezean** [comment]
please provide the first 8 chars of the submissions you want to compare, if you believe you've submitted the right things

i've downloaded the log of your latest 4 submissions and seems they are identical, since it's unlikely that two jars built from different source code would share a same size

<img width="297" alt="Screenshot 2023-12-28 at 03 05 26" src="https://github.com/hezean/sustc/assets/49837965/3cbf29c7-1947-4032-b541-efe686561fbc">

<img width="297" alt="Screenshot 2023-12-28 at 03 05 38" src="https://github.com/hezean/sustc/assets/49837965/0e2ddddf-3760-4b54-94dd-f550bb44c95d">

**Comment by dequeueing** [comment]
> please provide the first 8 chars of the submissions you want to compare, if you believe you've submitted the right things
> 
> i've downloaded the log of your latest 4 submissions and seems they are identical, since it's unlikely that two jars built from different source code would share a same size
> 
> <img alt="Screenshot 2023-12-28 at 03 05 26" width="297" src="https://private-user-images.githubusercontent.com/49837965/293086337-3cbf29c7-1947-4032-b541-efe686561fbc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM3Mjk3NzUsIm5iZiI6MTcwMzcyOTQ3NSwicGF0aCI6Ii80OTgzNzk2NS8yOTMwODYzMzctM2NiZjI5YzctMTk0Ny00MDMyLWI1NDEtZWZlNjg2NTYxZmJjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjI4VDAyMTExNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWZkMDU3MGRiNTBlNzE0ZjMyYmFhYjYzZTY2ZWU4NWUwMTM0ODJmNjEzMTA0MDRhMzU4MjJiNmU3OTcwMmQ0MGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.gBbgrJ9qLYD0MIpVskJ9qlREoxnsndFfEeOtCCsyysQ"> <img alt="Screenshot 2023-12-28 at 03 05 38" width="297" src="https://private-user-images.githubusercontent.com/49837965/293086365-0e2ddddf-3760-4b54-94dd-f550bb44c95d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDM3Mjk3NzUsIm5iZiI6MTcwMzcyOTQ3NSwicGF0aCI6Ii80OTgzNzk2NS8yOTMwODYzNjUtMGUyZGRkZGYtMzc2MC00YjU0LTk0ZGQtZjU1MGJiNDRjOTVkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjI4VDAyMTExNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTRjZGQ0MjMxNjgyY2ExNzY2MmQ3YTAxZTYyMDRjNjc4OGY0MjA0ZTVkYzEyZjE5Nzg2OGNhYjFmZWRiYTUxOGQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.v4caSZ9ez0Rjhf1VfiToJ2LXu08SwDizKidlCKrm1BA">

**Task ID**
Latest versoin:  [3af17daf]
Earlier version: [73f0ae3d]

If these two submissions are identical, there must be some problems with the `submitJar` application. 

Thank you.

**Comment by hezean** [comment]
yes, these two submissions have different jar files, you may
1. download the latest small dataset and run again
2. add some `System.out.print` or `log.fatal` in your service method to print the arguments and return values, then check the oj's `benchmark-stdout.log` attachment

**Comment by dequeueing** [comment]
> yes, these two submissions have different jar files, you may
> 
> 1. download the latest small dataset and run again
> 2. add some `System.out.print` or `log.fatal` in your service method to print the arguments and return values, then check the oj's `benchmark-stdout.log` attachment

Here is what I have done:

- Download the latest small dataset and run again. 
  Result: still get correct feedback, no `debug` record in the log.
- Add some System.out.print in my method (it seems that log.fatal not supported...). 
  Result: I can see the sout message in local benchmark, but not in the OJ's `benchmark-stdout.log`. (I am sure I submitted the correct jar)

![image](https://github.com/hezean/sustc/assets/127177415/2dd352c2-f3d7-4f5f-8169-3eb6c1b9e611)

![image](https://github.com/hezean/sustc/assets/127177415/1848cb07-661a-49ec-9bec-eb2d215e7d80)

![image](https://github.com/hezean/sustc/assets/127177415/960b645b-60ec-4f69-a4fa-ce8eca173914)

If I cannot get practical in feedback in local benchmark, I cannot improve my method. Thanks.

**Comment by hezean** [comment]
that's weird.. @WhatWEat is it possible for you to take some time tracking on it? i'm having an urgent task and hardly have time recently, many thanks

**Comment by hezean** [comment]
@dequeueing by the way, can you ask your teammate (if there's one) to submit the work (build jar, zip, and upload to oj) and see compare the report and logs with your submission? we may be hard to fix this without more info since no any other has reported this issue

**Comment by dequeueing** [comment]
I think one of my problems is that I cannot see the debug info in my terminal though the benchmark is incorrect. 

![image](https://github.com/hezean/sustc/assets/127177415/8b8dfc1e-8459-4fd9-a8fa-40a21851c279)


![image](https://github.com/hezean/sustc/assets/127177415/9321c973-7243-42af-ba95-f05a8862d3ff)

My `application.yml`:

![image](https://github.com/hezean/sustc/assets/127177415/621d67e1-99d4-4bef-80b5-e835a9da0659)

Another problem is that, in some cases I get all correct result in local benchmark,but oj fails. This makes me feeling hard where to debug.

Thank you.

**Comment by WhatWEat** [comment]
are you use the professional version of IDEA?

**Comment by hezean** [comment]
@dequeueing please confirm that the sql file you submitted could properly init the db, we found that some students update their table/function definitions in ide's console, then forget to update the sql file. if the problem still exists, please send your code to me by email for a further inspection

**Comment by dequeueing** [comment]
> @dequeueing please confirm that the sql file you submitted could properly init the db, we found that some students update their table/function definitions in ide's console, then forget to update the sql file. if the problem still exists, please send your code to me by email for a further inspection

Thank you very much.
